docs/rest/customSearchExample.http

```diff
@@ -105,4 +112,24 @@ x-ms-use-partial-indices: true
 ### A search query showing the Composite custom search parameter with Use/Postcode pair.
 # The usecase would be to evaluate multiple values together in chained search queries
 GET https://{{hostname}}/Patient?address-use-postcode=home$74047
-Authorization: Bearer {{bearer.response.body.access_token}}
\ No newline at end of file
+Authorization: Bearer {{bearer.response.body.access_token}}
+
+### POST additional patient data to demonstrate sorting by a custom search parameter.
+# @name patientBundle
+POST https://{{hostname}}
+content-type: application/json
+Authorization: Bearer {{bearer.response.body.access_token}}
+
+< ./Data/CustomSearchPatientsAdditional.json
+
+### A search query showing sorting by a custom search parameter.
+GET https://{{hostname}}/Patient?_sort=CreatedDate
```

Comment:
Should this also filter for where CreatedDate exists? Otherwise there will be a lot of results without the sort field.